lua/lib/renderer.lua

```diff
@@ -279,8 +279,11 @@ end
 
 local M = {}
 
+function is_bufnr_valid(bufnr)
+  return vim.fn.bufexists(bufnr) == 1
+end
 function M.draw(tree, reload)
-  if not tree.bufnr then return end
+  if not is_bufnr_valid(tree.bufnr) then return end
```

Comment:
could you just add an empty line between these two function. Also declaring is_bufnr_valid as local to avoid conflicts :)